### PR TITLE
removing dos/windows line ends and replace them with unix ones

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

File: bin/ParseDarwinSamplesheet.sh [hunk 1]
```diff
@@ -197,6 +197,8 @@ then
 else
 	for darwinSamplesheet in "${sampleSheetsDarwin[@]}"
 	do
+		cat ${darwinSamplesheet} | sed 's/\r/\n/g' | sed "/^[\s,]*$/d" > "${darwinSamplesheet}.converted"
```
Comment (pneerincx) [inline bin/ParseDarwinSamplesheet.sh]: Missing quotes for `${darwinSamplesheet}` -> `"${darwinSamplesheet"`

**Review by pneerincx — CHANGES_REQUESTED** [review]
See inline comment.

File: bin/ParseDarwinSamplesheet.sh (hunk 1 above)
Comment (RoanKanninga) [reply bin/ParseDarwinSamplesheet.sh]: check, fixed

**Review by pneerincx — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -40,27 +40,30 @@ See separate README_v1.md for details on the (deprecated) version
...
-               v    ⎜ LFS  ⎜ The NGS_DNA pipeline is finished  ⎜>> + >>⎜ LFS  ⎜ The GAP pipeline is finished         ⎜ 
-               v    ⎜ prm* ⎜ the ngs VCF files are ready.      ⎜       ⎜ prm* ⎜ the array VCF files are ready.       ⎜
-               v    ⎝__________________________________________⎠       ⎝_____________________________________________⎠
-               v                                                                           
+                                                    v
+                                                    v <<<< 2: ConcordanceCheck.sh 
+                                                    v
    ⎛¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯⎞
-   ⎜ LFS  ⎜ Copies the nsg VCF and the array VCF file to tmp*.                                       ⎜<<<< 2: ConcordanceCheck.sh 
+   ⎜ LFS  ⎜ Copies the nsg VCF and the array VCF file to tmp*.                                       ⎜
```
Comment (marieke-bijlsma) [inline README.md]:
```suggestion
   ⎜ LFS  ⎜ Copies the ngs VCF and the array VCF file to tmp*.                                       ⎜
```

**Review by marieke-bijlsma — DISMISSED** [review]

**Review by marieke-bijlsma — APPROVED** [review]

**Pull request merged**